X:7(a): Submission validity criteria (#39)
open-contracting-extensions/government-procurement-agreement: docs/gpa.md

        <td markdown=1>

* For *the procurement, including the nature and the quantity of the goods or services to be procured or, where the quantity is not known, the estimated quantity*, follow the guidance for [VII:2(b)](<#VII:2(b)>)

> * For *any requirements to be fulfilled, including any technical specifications, conformity assessment certification, plans, drawings or instructional materials*, the OCDS currently doesn't have structured criteria for submission validity

Reviewer: I think we want to say:

```suggestion
* For *any requirements to be fulfilled, including any technical specifications, conformity assessment certification, plans, drawings or instructional materials*, append to `tender/description`.
```

So that the information isn't simply discarded.
